build.gradle

```diff
@@ -193,7 +193,9 @@ subprojects {
 				logger.error("ERROR: Test: " + descriptor + " produced resource leak: " + event.message)
 			}
 		}
+	}
 
+	project.tasks.withType(Test).all {
```

Comment:
sorry I should have been clearer: there's already such a block on line 163 above 😄

Reply:
oops
fixed